.azure-pipelines-templates/install_deb.yml

```diff
@@ -60,6 +64,8 @@ steps:
   - ${{ if eq(parameters.target, 'SNPCC') }}:
       - script: |
           set -ex
+          export CC=$(command -v clang-12)
+          export CXX=$(command -v clang++-12)
```

Comment:
I think it'd be nicer to have this logic in `test_install_build.sh` so that the script is self-contained and doesn't depend on external external environment.

Reply:
Is it something like `TYPE=sgx ./test_install_build.sh <cmake args>` ?
I can't add additional argument because arguments are passed to cmake as `$@`.

Reply:
Or just trying to detect whether `clang-12` exists and if it does, use it. Otherwise `clang-10`?